Examples/Install-Win32OpenSSH/Install-Win32OpenSSH.ps1

```diff
@@ -0,0 +1,173 @@
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+$ErrorActionPreference = 'stop'
+$ProgressPreference = 'silentlycontinue'
+$VerbosePreference = 'continue'
+ 
+$WorkDir = 'c:\Amazon'
+if (!(Test-Path -Path $WorkDir)) {
+    mkdir -Path $WorkDir
+}
+Set-Location -Path c:\Amazon
+ 
+function Write-Log {
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory = $true)]
+    [string] $Message
+  )
+  Add-Content -Path $env:SystemDrive\Win32-OpenSSH.log -Value ('{0}: {1}' -f (Get-Date -Format o), $Message)
+}
+ 
+function Install-PowerShellCore {
+    Write-Log -Message 'Installing PowerShell Core'
+    $SourceURL = 'https://github.com/PowerShell/PowerShell/releases/download/v6.0.0/PowerShell-6.0.0-win-x64.zip'
+    $UnzipFolder = '{0}\pwsh' -f $env:SystemDrive
+    $TargetFile = '{0}\pwsh.zip' -f $UnzipFolder
+ 
+    if (Get-Command -Name pwsh.exe -ErrorAction Ignore) {
+        Write-Log -Message 'Skipping install of PowerShell Core. Already present.'
+        return
+    }
+    
+    mkdir -Path $UnzipFolder -ErrorAction Ignore
+    Invoke-WebRequest -OutFile $TargetFile -Uri $SourceURL
+    Expand-Archive -Path $TargetFile -DestinationPath $UnzipFolder -Force
+    Remove-Item -Path $TargetFile
+    [System.Environment]::SetEnvironmentVariable('PATH', ($env:PATH + (';{0}' -f $UnzipFolder)), [System.EnvironmentVariableTarget]::Machine)
+    Write-Log -Message 'Finished installing PowerShell Core'
+}
+ 
+function Install-PuTTY {
+  if (Get-Command -Name putty.exe -ErrorAction Ignore) {
+    Write-Log -Message 'Skipping install of PuTTY. Already installed.'
+    return
+  }
+ 
+  Write-Log -Message 'Installing PuTTY'
+  $PuTTYURL = 'https://the.earth.li/~sgtatham/putty/latest/w64/putty-64bit-0.70-installer.msi'
+  $ZipFile = '{0}\putty.msi' -f $pwd
+  Invoke-WebRequest -Uri $PuTTYURL -OutFile $ZipFile
+  msiexec /package "$ZipFile" /passive /norestart
+  Remove-Item -Path $ZipFile
+  Write-Log -Message 'Finished installing PuTTY'
+}
+ 
+ 
+function Install-OpenSSH {
+    ## https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH
+ 
+    $OpenSSHSourceURL = 'https://github.com/PowerShell/Win32-OpenSSH/releases/download/v1.0.0.0/OpenSSH-Win64.zip'
+    $OpenSSHZip = '{0}\OpenSSH-Win64.zip' -f $env:TEMP
+    $TargetPath = '{0}\OpenSSH' -f $env:ProgramFiles
+ 
+    Invoke-WebRequest -OutFile $OpenSSHZip -Uri $OpenSSHSourceURL
+    Unblock-File -Path $OpenSSHZip
+    Expand-Archive -Path $OpenSSHZip -DestinationPath $env:ProgramFiles -Force
+    Remove-Item -Path $OpenSSHZip -Force
+ 
+    Rename-Item -Path ('{0}-Win64' -f $TargetPath) -NewName $TargetPath -Force -ErrorAction Ignore
+ 
+    Set-Location -Path $TargetPath
+ 
+    powershell -ExecutionPolicy Bypass -File ('{0}\install-sshd.ps1' -f $TargetPath)
+    Write-Log -Message 'Installed SSHD'
+ 
+    ### Fix file permissions for host keys
+    & ./FixHostFilePermissions.ps1 -Confirm:$false
```

Comment:
File ````FixHostFilePermissions.ps1```` is not included in the PR

Reply:
It's not supposed to be included in the PR. It's part of the Win32-OpenSSH package.